SUIApp/SUIApp/Views/TabBar/AccountView.swift

```diff
@@ -0,0 +1,105 @@
+// AccountView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран пользователя
+struct AccountView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let avatarImageNameText = "avatar"
+        static let userNameText = "Nika"
+        static let cityNameText = "Ekaterinburg"
+        static let mappinImageSystemNameText = "mappin"
+        static let notificationTitleText = "Notification"
+        static let emptyText = ""
+        static let headerGradientHeightNumber: CGFloat = 10
+    }
+
+    // MARK: - Public Property
+
+    var body: some View {
+        navigationView
```

Comment:
```suggestion
        navigationView
```
Сложная структура вложенности, а в боди только один navigation/ Сложно читаемая